README.md

```diff
@@ -74,6 +74,8 @@ Troubleshooting issues with the plugin can be performed using these [tips](/docs
 - For upgrading the plugin from older version to the current released version, user needs to unmount all the volumes and follow the standard
  upgrade procedure described in docker guide. 
  
+- Encryption only supports in plugin version 3.1.1 and 3.3.1.
```

Comment:
@bhagyashree-sarawate Update this to:
Encryption not working with plugin version 3.2 and 3.3.